samples/nextjs/src/components/GraphQL-ConnectedDemo/index.tsx

```diff
@@ -144,4 +145,24 @@ export const getServerSideProps: GetServerSideComponentProps = async (rendering,
   return result.data;
 };
 
+/**
+ * Will be called during editing
+ * @param rendering {ComponentRendering}
+ * @param layoutData {LayoutServiceData}
+ * @param context {NextPageContext}
```

Comment:
@ambrauer Probably, need to place `type` before param name

Reply:
Ah yes, fixed